app/scripts/app-init.js

```diff
@@ -1,11 +1,12 @@
 // This file is used only for manifest version 3
 
-// Represents if importAllScripts has been run
 // eslint-disable-next-line
-let scriptsLoaded = false;
+const WORKER_KEEP_ALIVE_MESSAGE = 'UI_OPEN';
 
+// Represents if importAllScripts has been run
```

Comment:
I don't understand this comment with respect to `testMode`. Should we remove this? 
```suggestion
```